src/contracts/interfaces/IDelayedWithdrawalRouter.sol

```diff
@@ -39,8 +39,11 @@ interface IDelayedWithdrawalRouter {
     /// @notice Getter function for the mapping `_userWithdrawals`
     function userWithdrawals(address user) external view returns (UserDelayedWithdrawals memory);
 
+    /// @notice Getter function to get all delayedWithdrawals of the `user`
+    function getUserDelayedWithdrawals(address user) external view returns (DelayedWithdrawal[] memory);
+
     /// @notice Getter function to get all delayedWithdrawals that are currently claimable by the `user`
-    function claimableUserDelayedWithdrawals(address user) external view returns (DelayedWithdrawal[] memory);
+    function getClaimableUserDelayedWithdrawals(address user) external view returns (DelayedWithdrawal[] memory);
```

Comment:
needs correct comments on the interface too

Reply:
seems like the comments on the interface are correct?

Reply:
ahh mb it's just missing the one note about naming

Reply:
oh oh I forgot to mention, we're going to change the function name on the UI.  Chris Moller confirmed it wasn't a big change.

Reply:
OK, sounds like @gpsanant should be the final approver on this then if you've agreed on something offline